Lib/abc.py

```diff
@@ -28,7 +28,13 @@ def my_abstract_method(self, ...):
 class abstractclassmethod(classmethod):
     """A decorator indicating abstract classmethods.
 
-    Deprecated, use 'classmethod' with 'abstractmethod' instead.
+    Deprecated, instead use `classmethod` on top of `abstractmethod`:
+
+        @classmethod
+        @abstractmethod
+        def your_function(cls):
```

Comment:
Let's use the same function names as in the docs:
```suggestion
        def my_abstract_classmethod(cls, ...):
```